### PR TITLE
Add test_engine_crash_during_restore() skeleton

### PR DESCRIPTION
**Pull request by shuo-wu**
longhorn/longhorn#1336

### CODE REVIEW

**Comment by meldafrawi** [comment]
LGTM

File: manager/integration/tests/test_ha.py [hunk 1]
```diff
@@ -836,3 +836,28 @@ def test_rebuild_with_inc_restoration(
     delete_and_wait_pod(core_api, dr_pod_name)
     delete_and_wait_pvc(core_api, dr_pvc_name)
     delete_and_wait_pv(core_api, dr_pv_name)
+
+
+@pytest.mark.skip
```
Comment (yasker) [inline manager/integration/tests/test_ha.py:841]: `@pytest.mark.skip(reason="TODO")`

File: manager/integration/tests/test_ha.py [hunk 2]
```diff
@@ -836,3 +836,28 @@ def test_rebuild_with_inc_restoration(
...
+def test_engine_crash_during_restore(client, core_api, volume_name, pod_make):  # NOQA
```
Comment (yasker) [inline manager/integration/tests/test_ha.py:842]: Remove fixtures.

**Comment by shuo-wu** [comment]
The test plan is modified based on the merged PR.
And one more test case for DR volumes is introduced.

**Pull request merged**